### ISSUE TITLE
System rules have gotten less specific

### ISSUE THREAD

**Issue by ltGuillaume** [henrypp/simplewall#527]
I'd like to know why some system rules got less specific with this commit: https://github.com/henrypp/simplewall/commit/29e6ad97de3d16e7a7f4427102efb6d0522f20ee

I understand that System needs access, but now these ports are opened up for _all_ services.

**Comment by henrypp** [comment]
Same as changed DNS & DHCP rule earlier: dnscache service may disabled, but services can work over dns.

**Comment by ltGuillaume** [comment]
@henrypp Could you point me to some more info on this? Because I don't understand your explanation yet.

**Comment by ltGuillaume** [comment]
Could you please help me out here? I really don't understand what you meant by "services can work over dns" in relation to giving all services access to the RDP, SSDP and UPnP ports.
